src/field/field_door.c

```diff
@@ -6,16 +6,397 @@
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_22[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/22/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/22/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/22/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_23[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/23/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/23/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/23/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_24[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/24/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/24/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/24/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_25[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/25/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/25/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/25/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_26[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/26/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/26/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/26/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_27[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/27/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/27/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/27/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_28[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/28/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/28/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/28/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_29[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/29/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/29/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/29/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_30[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/30/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/30/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/30/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_31[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/31/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/31/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/31/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_32[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/32/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/32/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/32/2.4bpp"),
+};
+
+asm(".space 32");
+
+const u8 DoorAnimTiles_33[][0x100] =
+{
+    INCBIN_U8("graphics/door_anims/33/0.4bpp"),
+    INCBIN_U8("graphics/door_anims/33/1.4bpp"),
+    INCBIN_U8("graphics/door_anims/33/2.4bpp"),
+};
+
+asm(".space 32");
+
+static const struct DoorAnimFrame gDoorOpenAnimFrames[] =
+{
+    {4, -1},
+    {4, 0},
+    {4, 0x100},
+    {4, 0x200},
+    {0, 0},
+};
+
+static const struct DoorAnimFrame gDoorCloseAnimFrames[] =
+{
+    {4, 0x200},
+    {4, 0x100},
+    {4, 0},
+    {4, -1},
+    {0, 0},
+};
+
+const u8 DoorAnimPalettes_830F8A4[] = {1, 1, 1, 1, 1, 1, 1, 1}; // door 00
```

Comment:
Are these palettes or palette ids?  If they are palette ids, that should be reflected in their name to avoid confusion.